guide/12-enrich-data-with-thematic-information/part3_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>With nearly 20,000 demographic <strong><em>variables </em></strong>available, we can take advantage of data collections to narrow our variable search to at least relevant <strong><em>variables </em></strong>to begin with. Using the <code>unique</code> Pandas Series method, we can retrieve the data collections available and select from these to begin our search.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>